src/gui/transferlistwidget.cpp

```diff
@@ -158,6 +158,8 @@ TransferListWidget::TransferListWidget(QWidget *parent, MainWindow *main_window)
     editHotkey = new QShortcut(QKeySequence("F2"), this, SLOT(renameSelectedTorrent()), 0, Qt::WidgetShortcut);
     deleteHotkey = new QShortcut(QKeySequence::Delete, this, SLOT(softDeleteSelectedTorrents()), 0, Qt::WidgetShortcut);
     permDeleteHotkey = new QShortcut(QKeySequence("Shift+Delete"), this, SLOT(permDeleteSelectedTorrents()), 0, Qt::WidgetShortcut);
+    doubleClickHotkey = new QShortcut(QKeySequence("Return"), this, SLOT(torrentDoubleClicked()), 0, Qt::WidgetShortcut);
```

Comment:
Should i change all the others as well while i'm at it? Or there is some specific that i shouldn't?